### ISSUE TITLE
Issue importing field/column using BigDecimal format

### ISSUE THREAD

**Issue by nonpluseternal** [mariodavid/cuba-component-data-import#61]
Hello Mario,

Neat tool - thanks for your contribution.

I'm unable to import a BigDecimal-formatted field. Please take a look if you have time, I have some info below that could help you fix this:

It appears BigDecimal format is not handled in the getValue method for the DatatypeAttributeBinder.groovy file (dataimport-core-0.2.0.jar/de.diedavids.cuba.dataimport/binding/DataTypeAttributeBinder.groovy):

```
private getValue(AttributeBindRequest bindRequest) {
        switch (bindRequest.metaProperty.javaType) {
            case Integer: return getIntegerValue(bindRequest.rawValue, bindRequest.dataRow)
            case Double: return getDoubleValue(bindRequest.rawValue, bindRequest.dataRow)
            case Date: return getDateValue(bindRequest.importConfiguration, bindRequest.rawValue)
            case Boolean: return getBooleanValue(bindRequest.importConfiguration, bindRequest.rawValue, bindRequest.dataRow)
            case String: return getStringValue(bindRequest.rawValue)
        }
    }
```

**Comment by mariodavid** [comment]
Hi,

Thanks for reaching out.
Can you share some input values? Which values would you wanted to get parsed. Whats about deliminators etc?

Bye
Mario

**Comment by nonpluseternal** [comment]
For testing I used a simple csv file, importing to an entity with a string Name and BigDecimal Num(non-mandatory field. If I made the field mandatory it threw an exception related to AttributeBinder)

*example.csv*
Name,Num
Test,14

I expected it to import both values. What actually happened was, it did not show success, it imported the string Name correctly and left "Num" (BigDecimal) blank.

**Comment by mariodavid** [comment]
Right, bigdecimal is not supported as you said.

I was just thinking about that about:
12.3
1,456,900.758813

As well as situations where you have localisations in place (e.g in DE it would be 1.987.654,65008)

I thought you have some additional edge cases for me :)

**Comment by mariodavid** [comment]
btw:
there is always the escape path in case the app component does not have support for specific datatypes to bind: [Custom attribute binding script](https://github.com/mariodavid/cuba-component-data-import#custom-attribute-binding-script).

Something like this should do the trick:
```
return new BigDecimal(rawValue)
```

But I will plan native support for BigDecimal somewhere 0.3.0 / 0.4.0.

**Comment by nonpluseternal** [comment]
Excellent - thank you!

**Comment by mariodavid** [comment]
we can actually leave it open, so that I can use it to keep track of the actual implementation.

**Comment by mariodavid** [comment]
done in #61. Will be released in 0.3.0. Thanks for pointing it out.
